src/node/Launcher.ts

```diff
@@ -190,6 +190,9 @@ class ChromeLauncher implements ProductLauncher {
       '--enable-automation',
       '--password-store=basic',
       '--use-mock-keychain',
+      '--enable-blink-features=IdleDetection',
+      // TODO sadym: remove '--enable-blink-features=IdleDetection'
+      // after IdleDetection is turned on by default
```

Comment:
nit: TODOs go before the line they're commenting on

```suggestion
      // TODO(sadym): remove '--enable-blink-features=IdleDetection'
      // once IdleDetection is turned on by default.
      '--enable-blink-features=IdleDetection',
```